### PR TITLE
Add Google Analytics tag to the pages

### LINKED ISSUE #699: Add google analytics to sef-site
**Is your feature request related to a problem? Please describe.**
We need to add google analytics for the sef-site to get analytical information of the site

**Describe the solution you'd like**
Add google analytics to the site by using the google analytics tag

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by Piumal1999**
## Purpose

The purpose of this PR is to fix #699

## Goals

Adding google analytics to sef-site

## Approach

Added the google analytics tag to all pages

### Screenshots

N/A

### Preview Link

https://pr-700-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Related PRs
 
N/A

## Test environment
 
N/A

## Learning

https://support.google.com/analytics/answer/1008015?hl=en

### CODE REVIEW

**Review by anjula-sack — DISMISSED** [review]

File: src/main/webapp/apply.html [hunk 1]
```diff
@@ -2,6 +2,17 @@
 <html>
 
 <head>
+
```
Comment (jayasanka-sack) [inline src/main/webapp/apply.html]: Remove this white space

File: src/main/webapp/apply.html [hunk 2]
```diff
@@ -2,6 +2,17 @@
...
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-167873271-1"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+
```
Comment (jayasanka-sack) [inline src/main/webapp/apply.html]: Remove this white space

File: src/main/webapp/apply.html [hunk 3]
```diff
@@ -2,6 +2,17 @@
...
+        gtag('config', 'UA-167873271-1');
+    </script>
+
```
Comment (jayasanka-sack) [inline src/main/webapp/apply.html]: Remove this white space

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Nice job @Piumal1999 !

Could you please remove those whitespace?

File: src/main/webapp/apply.html (hunk 3 above)
Comment (Piumal1999) [reply src/main/webapp/apply.html]: Done

**Comment by Piumal1999** [comment]
Removed whitespaces

**Review by jayasanka-sack — APPROVED** [review]
Thanks @Piumal1999 !

**Review by anjula-sack — APPROVED** [review]
Awesome! @Piumal1999  👏

**Pull request merged**